connectors/bigtable/flink-connector-gcp-bigtable/src/main/java/com/google/flink/connector/gcp/bigtable/internal/writer/BigtableSinkWriter.java

```diff
@@ -0,0 +1,63 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.google.flink.connector.gcp.bigtable.writer;
+
+import org.apache.flink.api.connector.sink2.SinkWriter;
+
+import com.google.flink.connector.gcp.bigtable.serializers.BaseSerializer;
+
+import java.io.IOException;
+import java.util.concurrent.ExecutionException;
+
+/**
+ * Base writer class.
```

Comment:
is it really a base writer class?

Reply:
Entry class for the writer may be better? This is the entry point to the actual writer which is FlushableWritter

Reply:
yeah that's more descriptive